packages/core/src/html/warnings.ts

```diff
@@ -29,6 +29,7 @@ function _warnConflictingAttributes(node: MbNode, attribute: string, attrsConfli
  * @param element Root element to check
  * @param namePairs Object of slot name pairs with each pair in the form deprecated : correct
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function _warnDeprecatedSlotNames(element: MbNode, namePairs: { [name: string]: string }) {
```

Comment:
We should be able to remove this function entirely as well! 🎉 
If we need it again (hopefully never), we can recover it from the history :)